newbank/server/NewBank.java

```diff
@@ -455,6 +342,10 @@ private String getHelpPayBackLoan() {
         return "PAYBACKLOAN <Loan Number>\t\t\t\t\t\t\tThe loan is repaid with interest\n";
     }
 
+    private String getHelpPartPayBackLoan() {
```

Comment:
This needs adding to the getHelp method above, and to the getHelp test

Reply:
Done.

Reply:
Good job @andehbrown and @kimoshinowo.